src/main/java/org/dcsa/ebl/service/impl/ShippingInstructionServiceImpl.java

```diff
@@ -504,7 +504,7 @@ private Mono<ShipmentEvent> shipmentEventFromShippingInstruction(
         shippingInstruction.getShippingInstructionUpdatedDateTime());
   }
 
-  static Mono<ShipmentEvent> getShipmentEventFromShippingInstruction(
+  private static Mono<ShipmentEvent> getShipmentEventFromShippingInstruction(
```

Comment:
can we also remove the static since this is the only static method we have, so it falls out of place